docs/api/webview-tag.md

```diff
@@ -343,6 +343,25 @@ Returns `boolean` - Whether the guest page can go to `offset`.
 
 Clears the navigation history.
 
+#### `<webview>.length()`
```

Comment:
Are these actually exposed on webview too? I think there's an allow list of methods or something somewhere iirc.  Also wouldn't be too upset if we just didn't expose these on webviews and made people use the `webContents` API themselves

Reply:
Yes agreed on the second point, we should just not expose this on the webview side. I deleted the endpoint and tests but forgot to update this doc. Will delete!